extractors/generating_sns_products/aggregate_flows_from_google_data.py

```diff
@@ -0,0 +1,172 @@
+"""
+This script will create the movement multipliers using:
+- A populations estimate for each local authority in Scotland from 2018 (mid year) https://www2.gov.scot/Resource/0046/00462936.csv
+- Google mobility data giving the number of movements between geographic regions in 2020 https://www.gstatic.com/covid19/mobility/Global_Mobility_Report.csv
+- A table mapping the iso_3166_2 name of each local authority to the local authority name and code
+   compiled by hand, Jess Enright, 30 June 2020
+Python requirements:
+- pathlib
+- pandas
+- zipfile
+- urllib.request
```

Comment:
I think pathlib, zipfile and urllib are part of the python standar libraries, so we don't really need to list them.